src/components/utils.ts

```diff
@@ -1,7 +1,7 @@
 export const bytesToSize = (bytes: number): string => {
   if(bytes === 0) return '0 B';
   const sizes = ["Bytes", "KB", "MB", "GB", "TB"];
```

Comment:
Just a nitpick, but I would to this instead to be precise:
```suggestion
  const sizes = ["Bytes", "KiB", "MiB", "GiB", "TiB"];
```